lib/widgets/home_body_content.dart

```diff
@@ -44,14 +44,17 @@ class HomeBodyContent extends StatelessWidget {
 
   List<Widget> buildList(AsyncSnapshot snapshot) {
     return List.generate(snapshot.data.results.length, (index) {
+      print(snapshot.data.results[index]);
```

Comment:
Just as a good practice, don't leave prints on the code, if you really want to log this at a development level lets try and use this: assert(() { debugPrint(snapshot.data.results[index]) }());

Reply:
It was only to fix an error. Removed.